### DISCUSSION TITLE
Code template activation

### DISCUSSION THREAD

**Discussion by adidiot** [dbeaver/dbeaver#22473] [Q&A]
### Description

great tool thanks!
I use the last version of dbeaver CE on Ubuntu 23.10.
It seems code template does not work
After typing **swhere** and **ctrl-space** only the "no proposals" window appears.
Thanks for support

### DBeaver Version

DBeaver CE 23.3.3

### Operating System

Ubuntu 23.0

### Database and driver

db2, Oracle, ...

**Comment by LonwoLonwo** [comment]
Hello @adidiot 

Did you try to use ctrl+alt+space? 
https://github.com/dbeaver/dbeaver/wiki/SQL-Templates

**Comment by adidiot** [comment]
Yes it works, I use to use **ctrl+space**  like in Eclipse ..
Many Thanks for your quick answer.
I close the ticket.
